recipes/busybox/busybox-1.24.1/defconfig

```diff
@@ -554,24 +608,9 @@ CONFIG_HEXDUMP=y
 # CONFIG_LOSETUP is not set
 # CONFIG_LSPCI is not set
 # CONFIG_LSUSB is not set
-# CONFIG_MDEV is not set
-# CONFIG_FEATURE_MDEV_CONF is not set
-# CONFIG_FEATURE_MDEV_RENAME is not set
-# CONFIG_FEATURE_MDEV_RENAME_REGEXP is not set
-# CONFIG_FEATURE_MDEV_EXEC is not set
-# CONFIG_FEATURE_MDEV_LOAD_FIRMWARE is not set
 # CONFIG_MKSWAP is not set
 # CONFIG_FEATURE_MKSWAP_UUID is not set
 CONFIG_MORE=y
-# CONFIG_MOUNT is not set
-# CONFIG_FEATURE_MOUNT_FAKE is not set
-CONFIG_FEATURE_MOUNT_VERBOSE=y
```

Comment:
You should take care to keep all these CONFIG_FEATURE_MOUNT_* options in.

Or better, fixup the busybox_mount USE flag specified in busybox-configure.inc, to enable them that way.